app/javascript/gobierto_dashboards/webapp/Maker.vue

```diff
@@ -87,91 +87,122 @@ export default {
       item: null,
       configuration: null,
       viewerLoaded: false,
-      publicLabel: I18n.t("gobierto_dashboards.public") || "",
+      publicLabel: I18n.t("gobierto_dashboards.public") || ""
     };
   },
   computed: {
     id() {
       return this.$root.$data?.id;
     },
     title() {
-      return this.configuration?.attributes?.title || I18n.t("gobierto_dashboards.default_title") || "";
+      return (
+        this.configuration?.attributes?.title ||
+        I18n.t("gobierto_dashboards.default_title") ||
+        ""
+      );
     },
     status() {
       return this.configuration?.attributes?.visibility_level === "active";
-    },
+    }
   },
   created() {
     this.getConfiguration();
   },
   mounted() {
-    document.addEventListener('dragover', this.dragoverPosition)
+    document.addEventListener("dragover", this.dragoverPosition);
   },
   destroyed() {
-    document.removeEventListener('dragover', this.dragoverPosition)
+    document.removeEventListener("dragover", this.dragoverPosition);
   },
   methods: {
     async getConfiguration() {
-      ({ data: this.configuration } = this.id ? await this.getDashboard(this.id) : { data: {} });
-      this.dirty = false
+      ({ data: this.configuration } = this.id
+        ? await this.getDashboard(this.id)
+        : { data: {} });
+      this.dirty = false;
     },
     dragoverPosition({ clientX, clientY }) {
-      this.x = clientX
-      this.y = clientY
+      // current mouse position
+      this.x = clientX;
+      this.y = clientY;
+    },
+    debounce(func, timeout) {
```

Comment:
Not a big deal, but could this be a directive for easly reuse it? It's a common pattern

Reply:
Yeah, actually I was thinking about it, but I did'nt know where to place it, as it's not a vue-filter, not a mixin... I'm more inclined to [directive](https://es.vuejs.org/v2/guide/custom-directive.html), but look at the common declarations... I've to investigate further